File: Practice/src/introduction_coding_tests/Ranking.java
package introduction_coding_tests;

public class Ranking {
//	등수 매기기
	public int[] solution(int[][] score) {
		int[] answer = new int[score.length];
		for (int i = 0; i < answer.length; i++) {
			for (int j = 0; j < answer.length; j++) {
				if (score[i][0] + score[i][1] < score[j][0] + score[j][1]) {
					answer[i]++;
				}
			}
			answer[i]++;
		}
		return answer;
	}
}

File: Practice/src/introduction_coding_tests/Ranking.js
function solution(score) {
    score = score.map((i) => i[0] + i[1]);
    const sort = [...score].sort((a, b) => b - a);
    return score.map((i) => sort.indexOf(i) + 1);
    // return score.map(
    //     (el) => score.filter((v) => v[0] + v[1] > el[0] + el[1]).length + 1
    // );
}
